usagov/usagov-2021
# USAGOV-106-Fix-Scams-Wizard-Card

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-106

## Description

This fixes a typo that was preventing the card for the scams wizard from being able to pull fields from the node, so it would fall back to using fields from the menu item.
This should also fix [USAGOV-1034](https://cm-jira.usa.gov/browse/USAGOV-1034) Life events cards: Images are omitted

## How to Test
Edit the menu item for `Where to report a scam`
Set the link field to `Where to report scams (823)`
Visit the scams nav page (`/scams-and-fraud`)
Confirm that the card links to the scams wizard (`/where-report-scams`)

Visit /life-events
Confirm that the cards are showing their images

## Checklist for the Developer



## Checklist for the Peer Reviewers
